src/makeCalculator.js

```diff
@@ -35,7 +35,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  // creation of our hero
+  const internalCalculator = {
+    result: 0,
+
+    operate: function(operation, number) {
```

Comment:
не обов'язкого писати function
```suggestion
    operate(operation, number) {
```